pkg/sql/exec_util.go

```diff
@@ -187,7 +187,7 @@ var (
 	}
 	MetaTxnAbort = metric.Metadata{
 		Name:        "sql.txn.abort.count",
-		Help:        "Number of SQL transaction ABORT statements",
+		Help:        "Number of SQL transaction abort errors",
```

Comment:
This is the user-visible change I was thinking about. Maybe it's innocuous?

Reply:
Yeah, it's innocuous. The data isn't changing, I'm just fixing the help text to match the implementation (and is this help text even shown in the UI? I can't find it).